docker/.env

```diff
@@ -1,8 +1,8 @@
 # component versions
 
-ARANGO_DB_VERSION=3.9.6
+ARANGO_DB_VERSION=3.10.3
```

Comment:
What is the reason for upgrade to ArangoDb 3.10? Is it because of M1?
I don't mind upgrading as long as it's fully backward compatible, but Spline 0.7 and even 1.0 was developed and tested on ArangoDb 3.9, and there is a ticket for upgrade to 3.10 in the next Spline version 1.1 - https://github.com/AbsaOSS/spline/issues/1115